ee/WFLY-1160-Provide_ability_to_configure_a_global_directory_which_puts_the_contents_in_the_classpath.adoc

```diff
@@ -0,0 +1,86 @@
+= Provide ability to easily apply certain JBoss module libraries to all deployments running in a server
+:author:            Yeray Borges
+:email:             yborgess@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+Wildfly users can share libraries across all deployed applications without having to modify the packaging of their applications creating a custom module and making it global using the EE subsystem:
+
+  [standalone@localhost:9990 /] module add --name=com.test --absolute-resources=/Users/me/libs:/Users/me/libs/my-utils1.jar:/Users/me/libs/my-utils2.jar
```

Comment:
Can you please more highlights that "method add" not will be used in the final solution?

Reply:
Added:

> This command is not appropriate for use in a managed domain or when connecting to the management CLI remotely. Modules should be added and removed manually in a production environment.

> This RFE is opened to have an alternative new operation to address simpler cases

Hi @vmarekrh , would you mind to read it again and give me feedback? I think with those sentences is clear we are not going to use module add for this feature.

Reply:
Hi, @yersan I was discussing with my colleague and it is only my point of view.
It is ok now.